README.md

```diff
@@ -212,14 +230,41 @@ Behind the scenes, the extension builds the Pypi dependencies tree by running `p
       ```
 5. In the same terminal, install your python project and dependencies according to your project specifications.
 
-## .NET Projects
-For .NET projects which use NuGet packages as depedencies, the extension displays the NuGet depedencies tree, together with the information for each depedency.  
+### .NET Projects
+For .NET projects which use NuGet packages as dependencies, the extension displays the NuGet dependencies tree, together with the information for each dependency.  
 Behind the scenes, the extension builds the NuGet dependencies tree using the [NuGet deps tree](https://github.com/jfrog/nuget-deps-tree) npm package.
 
 Important:
-- Does your project define its NuGet depedencies using a *packages.config* file? If so, then please make sure the `nuget` CLI is installed on your local machine and that it is in your system PATH. The extension uses the `nuget` CLI to find the location of the NuGet packages on the local file-system.
+- Does your project define its NuGet dependencies using a *packages.config* file? If so, then please make sure the `nuget` CLI is installed on your local machine and that it is in your system PATH. The extension uses the `nuget` CLI to find the location of the NuGet packages on the local file-system.
 - The project must be restored using `nuget restore` or `dotnet restore` prior to scanning. After this action, you should click on the Refresh ![Refresh](resources/readme/refresh.png) button, for the tree view to be refreshed and updated.
 
+## The CI View
+The CI view of the extension allows you to view information about your builds directly from your CI system. This allows developers to keep track of the status of their code, while it is being built, tested and scanned as part of the CI pipeline, regardless of the CI provider used.
+
+This information can be viewed inside JFrog VS Code Extension, from the JFrog Panel, after switching to CI mode.
+
+The following details can be made available in the CI view.
+
+- Status of the build run (passed or failed)
+- Build run start time
+- Git branch and latest commit message
+- Link to the CI run log
+- Security information about the build artifacts and dependencies
+
+### How Does It Work?
+The CI information displayed in VS Code is pulled by the JFrog Extension directly from JFrog Artifactory. This information is stored in Artifactory as part of the build-info, which is published to Artifactory by the CI server. 
+
+Read more about build-info in the [Build Integration](https://www.jfrog.com/confluence/display/JFROG/Build+Integration) documentation page. If the CI pipeline is also configured to scan the build-info by JFrog Xray, the JFrog VS Code Extension will pull the results of the scan from JFrog Xray and display them in the CI view as well.
+
+Read the [Setting up CI Integration](https://www.jfrog.com/confluence/display/JFROG/JFrog+IntelliJ+IDEA+Plugin#JFrogIntelliJIDEAPlugin-SettingupCIintegration) for more information.
+
+### Displaying Build Information
```

Comment:
Displaying Build Information --> Setting Up the CI View
(let's also update the table of content).